torchvision/models/resnet.py

```diff
@@ -575,6 +575,13 @@ class Wide_ResNet101_2_Weights(WeightsEnum):
 def resnet18(*, weights: Optional[ResNet18_Weights] = None, progress: bool = True, **kwargs: Any) -> ResNet:
     """ResNet-18 from `Deep Residual Learning for Image Recognition <https://arxiv.org/pdf/1512.03385.pdf>`__.
 
+    .. note::
+       Bottleneck in torchvision places the stride for downsampling at 3x3 convolution(self.conv2)
+       while original implementation places the stride at the first 1x1 convolution(self.conv1)
+       according to "Deep residual learning for image recognition"https://arxiv.org/abs/1512.03385.
+       This variant is also known as ResNet V1.5 and improves accuracy according to
+       https://ngc.nvidia.com/catalog/model-scripts/nvidia:resnet_50_v1_5_for_pytorch.
```

Comment:
This is added on the wrong place. ResNet18 and 34 use the basic `BasicBlock` not `Bottleneck`. It's the 50, 101 and 152 versions of the model that use the `Bottleneck` class.

Reply:
Sorry, let me correct soon !